### PR TITLE
Frontend development

### PR DESCRIPTION
**Pull request by baloveless**
The team has completed the user information and change user information page, and there is a skeleton of the redirect page which will be completed on the next merge. Some tests need to be fixed, I won't merge the PR until those are sorted. This merge will leave only a handful of changes until MVP, the largest being the change password page. I also added some changes to the layout.

### CODE REVIEW

**Review by mackfoggia — DISMISSED** [review]
It looks like we're still picking up some .vs files, I would like to keep those out of the development branch if at all possible

**Comment by baloveless** [comment]
> It looks like we're still picking up some .vs files, I would like to keep those out of the development branch if at all possible

I added a line to the .gitignore, hopefully .vs folder wont be there in future

File: back/src/routes/airtable.ts [hunk 1]
```diff
@@ -105,6 +105,7 @@ airtableRouter.post('/signout', function(req, res, next) {
 
 
 airtableRouter.post("/getInfo", function (req, res) {
+  console.log(JSON.stringify(req.body[0]))
```
Comment (mackfoggia) [inline back/src/routes/airtable.ts]: should this be in the pr?

File: frontend/src/components/Redirect.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  const { id } = useParams()
+  const organizationName = "Brown Hope"
+  const applicationName = "Black Resilience Fund 2021 Application Form"
+  const interviewDate = "June 21"
+  const intakeCallsDate = "July 1"
+  const specialCases = "a food box or mutual aid"
+  // need to find right name for these variables. 
+  const infoURL = "https://www.blackresiliencefund.com/faq"
+  const contactEmail = ""
+  const contactURL = "https://www.brownhope.org/contact"
```
Comment (mackfoggia) [inline frontend/src/components/Redirect.tsx:14]: The appropriate contact address for this would be info@blackresiliencefund.com

Comment (mackfoggia) [reply frontend/src/components/Redirect.tsx:14]: Just realized that's an URL and not an email, but yeah Tracy would like this email to be present for questions about the application on this page and the pages that show status.

File: frontend/src/util/config.ts [hunk 3]
```diff
@@ -4,8 +4,8 @@ import dotenv from "dotenv"
  * above line is causing errors, needs to be resolved to use .env
  */
 //const server = process.env.EXTERNAL_IP;
-//const server = "localhost:5000";
-const server = '34.82.220.171';
+const server = "localhost:5000";
+//const server = '34.82.220.171';
```
Comment (mackfoggia) [inline frontend/src/util/config.ts]: I know it's a hassle but we need this to stay on the IP and not localhost for merge into development

File: back/src/routes/airtable.ts (hunk 1 above)
Comment (mackfoggia) [reply back/src/routes/airtable.ts]:
```suggestion
```

File: frontend/src/util/config.ts (hunk 3 above)
Comment (mackfoggia) [reply frontend/src/util/config.ts]: marking as needs work just in case

**Review by mackfoggia — CHANGES_REQUESTED** [review]

File: back/src/routes/airtable.ts (hunk 1 above)
Comment (baloveless) [reply back/src/routes/airtable.ts]: removed that line, that was to sort out some of my confusion

**Comment by baloveless** [comment]
I'm having an issue with the account information page. The route is returning status code 500. The same thing happens when I try the same with the old frontend

**Comment by baloveless** [comment]
> I'm having an issue with the account information page. The route is returning status code 500. The same thing happens when I try the same with the old frontend

That problem got resolved. This is ready to go I believe.

**Comment by baloveless** [comment]
Read last two commits for more details, but I think this is at MVP minus a few tests.

**Comment by mackfoggia** [comment]
If this is ready for merge, change the server to the IP instead of localhost and I will approve changes, everything looks great to me!

**Review by mackfoggia — APPROVED** [review]

**Pull request merged**